src/nvim/api/private/helpers.c

```diff
@@ -2078,3 +2081,17 @@ bool parse_float_config(Dictionary config, FloatConfig *fconfig, bool reconf,
   }
   return true;
 }
+
+/// Helper function tos et current_sctx for api calls
```

Comment:
"helper" is never a useful word in any docstring or symbol name. All code is helpful
```suggestion
/// Sets current_sctx for API calls.
```

Reply:
Agreed